content/doc/developer/internationalization/new-internationalization.adoc

```diff
@@ -0,0 +1,339 @@
+---
+title: New Internationalization
+layout: developersection
+---
+ 
+https://wiki.jenkins.io/display/JENKINS/How+to+view+Jenkins+in+your+language[How
+to view Jenkins in your language].
+
+'''''
+
+== Translations to Specific languages
+
+* https://wiki.jenkins.io/display/JENKINS/Chinese+%28Traditional%29+Translation[Chinese
```

Comment:
I think all the links in this section are broken.   Should we prioritize porting these to jenkins.io?

Reply:
Didn't found this particular file in https://github.com/jenkins-infra/docker-confluence-data/tree/main/content/JENKINS so yes this link is dead, but the others are up again.

For information, redirections to more pertinent pages on jenkins.io can also be added in https://github.com/jenkins-infra/docker-confluence-data/blob/main/nginx/nginx.conf if needed.